content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md

```diff
@@ -0,0 +1,71 @@
+
+This article provides instructions on how to move from using the deprecated dynamic DNS service to the new **ddns-updater** application. 
+
+The ddns-updater application is a lightweight universal dynanmic DNS (DDNS) updater with web UI. 
+When installed, a container launches with root privileges in order to apply the correct permissions to the ddns-updater directories. 
+Afterwards, the containter runs as a non-root user.
+
+## Before You Begin
+Before you configure the new ddns-updater application:
+
+* Disable the Dynamic DNS service.
+  Go to **System Settings > Services** and disable the service and clear the **Start Automatically** checkbox. 
+  This prevents the service from re-enabling after a system restart.
+
+* Review your Dynamic DNS service settings and note all provider, domain, IP address, port number, URL and credential (username and password) settings.
+
+If establishing a new provider, create the user account before proceeding. Otherwise, use the existing provider details.
+
+If you want to grant access to a specific user (and group) other than using the defaults, add the new non-root administrative user and take note of the UID and GID for this user.
+
+## Migrating from TrueNAS Dynamic DNS Service
+
+After disabling the dynamamic DNS service, install the ddns-updater application. Go to **Apps** click on **Available Applications** and locate the **ddns-updater** application widget.
+
+{{< trueimage src="/images/SCALE/22.12/ddnsUpdaterAppWidget.png" alt="ddns-updater Application Widget" id="1 ddns-updater Application Widget" >}}
+
+Click **Install** to open the **ddns-updater** configuration wizard.
+
+Select the timezone that applies to your server location from the **Timezone** dropdown list.
+
+Click **Add** to the right of **DNS Provider Configuration** to display provider setting options. 
+Select the DDNS provider from the **Provider** dropdown list. 
+Each provider displays the settings required to establish a connection with and authenticate to that provider. 
+
+{{< trueimage src="/images/SCALE/22.12/ddns-updaterDNSProviderConfigAdd.png" alt="ddns-updater Add DNS Provider Configuration" id="2 Add ddns-updater Provider Configuration" >}}
+
+{{< hint type=info >}}
+The SCALE service dynamic DNS list includes providers not on the ddns-Updater application list. 
+If the service configuration uses a service not in the application, create a new account with a provider on the list in the ddns-updater application.
+{{< /hint >}}
+
+The update period default for the SCALE service is different from the application. Accept the default settings.
+
+{{< trueimage src="/images/SCALE/22.12/ddns-updaterConfigPeriodSettings.png" alt="ddns-updater Provider Settings" id="3 Add ddns-updater Provider Settings" >}}
+
+Add the address(es) for the [shoutrrr gopher notification service(s)](https://containrrr.dev/shoutrrr/0.7/services/overview/) you want to use. Separate each entry with a comma.
```

Comment:
Is configuring shoutrrr notifications required or optional? If optional, maybe an _If... any_ would be better than _Add... the_

Reply:
It is unclear if it is optional so I included the instructions.